core/src/commonMain/kotlin/com/ustadmobile/core/util/SymLink.kt

```diff
@@ -0,0 +1,3 @@
+package com.ustadmobile.core.util
+
+expect fun createSymLink(oldPath: String, targetPath: String)
```

Comment:
This expect/actual can only be used by JVM and Android. I think the expect signature should be in commonJvmMain